app/mentoring/models.py

```diff
@@ -57,7 +56,10 @@ class Profile(models.Model):
     )  # TODO: this requires user uploading, may not be the best idea
     bio = models.TextField(max_length=1_000, blank=True)
     birthday = models.DateField(null=True, blank=True)
-    location = gis_models.PointField(geography=True, spatial_index=True)
+    location = models.PointField(
+        geography=True,
+        # spatial_index=True,
```

Comment:
spatial index is on by default, if you don't want it, you'll have to set it to False. 

Saying that I think we _really_ want the index. As it should help the query performance significantly, and this will be our many expense.

Reply:
Oh, this was an accident, I was just trying random things to try and get the web version to work